taxi/urls.py

```diff
@@ -1,4 +1,4 @@
-from django.urls import path
+from django.urls import path, include
```

Comment:
Why did you import include and didn't use it?

Reply:
Deleting